robot_assignments/web/jet2.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     String
+
+*** Variables ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${urlhome}      http://125.26.15.143:13132/home.html
+${username1}    helloworld
+${password1}    1234
+${username2}    oz4899
+${password2}    1234
+${loginIDFieldXpath}    //input[@id='username']
+${loginPasswordFieldXpath}      //input[@id='password']
+${loginButtonXpath}     //button[@type='submit']
+${homePageContentXpath}     //div[@class='container']/div[@class='box']
+${searchFieldXpath}     //div[@class='column is-four-fifths']/input 
+${searchButtonXpath}       //button[@id='searchbtn'] 
+${itemListXpath}        //div[@class='card-content p-4']/p
+${noResultFoundXpath}       //div[@class='box']/div/div
+${firstItemXpath}       //div[@class='box']/div/div[1]
+${addToCartButtonXpath}      //button[@class='button is-info is-size-5']
+${itemQuantityXpath}     //span[@id='lblCartCount']/b
+${plusButtonXpath}      //div[@id='remove']/button
+${cartHoverXpath}       //div[@class='column']/div/div[@style='text-align: center;']/div/div[@class='dropdown-trigger']
+${clearCartButtonXpath}     //button[@class='button is-danger']
+
+*** Keywords ***
+Open Website
+    Open Browser  url=${urlLogin}  browser=chrome
+
+Login Website   
+    Input Text   ${loginIDFieldXpath}    ${username2}
+    Input Text   ${loginPasswordFieldXpath}      ${password2}
+    Click Element   ${loginButtonXpath}
+    Wait Until Page Contains Element    ${homePageContentXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlhome}     ${currentUrl}
+
+*** Test cases ***
+Testcase 01 - Verify that user cannot login with wrong username
+    Open Website
+    Input Text   ${loginIDFieldXpath}     ${username1}
+    Input Text   ${loginPasswordFieldXpath}     ${password1}
+    Click Element   ${loginButtonXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlLogin}     ${currentUrl}
```

Comment:
There is no checking on error message as specify in test case ka. Please add

Verify that cannot access into system and user get failed message
![image](https://user-images.githubusercontent.com/83345045/157395218-97c7feff-a3f7-4af1-b623-549629c49b50.png)